### ISSUE TITLE
erroneous error: pkg_resources.require is not callable (not-callable)

### ISSUE THREAD

**Issue by mbdevpl** [PyCQA/pylint#780]
The following code shouldn't report any errors:

```python
"""
    Testing pkg_resources.require().
"""

import sys

import pkg_resources
pkg_resources.require('pylint>=1.5.3')

import pylint

sys.argv = ['pylint', '--version']
pylint.run_pylint()
```

It runs:
```
../experiments$ python3.5 pkg_resources_require_test.py
No config file found, using default configuration
pylint 1.5.3, 
astroid 1.4.3
Python 3.5.1 (default, Jan  6 2016, 17:28:46) 
[GCC 5.3.0 20151204]
.../experiments$
```

However:

```
.../experiments$ pylint -r n pkg_resources_require_test.py
No config file found, using default configuration
************* Module pkg_resources_require_test
E:  8, 0: pkg_resources.require is not callable (not-callable)
.../experiments$ 
```

But `pkg_resources.require` is callable:
https://pythonhosted.org/setuptools/pkg_resources.html#basic-workingset-methods

quote:
> Thus, you can use e.g. pkg_resources.require() as an abbreviation for (...)

another:
> require(*requirements)
>
>    Ensure that distributions matching requirements are activated

**Comment by The-Compiler** [comment]
Seems like it's [missing here in astroid](https://github.com/PyCQA/astroid/blob/master/astroid/brain/brain_stdlib.py#L151-L193). I'm guessing `run_script` and `iter_entry_points` could be added as well.

**Comment by PCManticore** [comment]
Anyone up for sending a PR? ;-)

**Comment by mbdevpl** [comment]
I could submit it after a brief read-through... In the meantime, just out of curiosity, why does the code regarding `pkg_resources` reside in file that has "stdlib" in the name?

**Comment by PCManticore** [comment]
To be honest, I don't know. But it can be moved out into its own file.

**Comment by mbdevpl** [comment]
Can you give me let's say 24h from now? I will check back before that time with one/few concrete good ideas or with "I can't do it". Is that OK? ;)

**Comment by PCManticore** [comment]
Sure. ;-)

**Comment by mbdevpl** [comment]
Great! Challenge accepted ;)

**Comment by mbdevpl** [comment]
I didn't come up with a nice way to extract the `pkg_resources` code to its own file. I know too little about astroid to do that well, so I will not attempt it. But, I've added 3 functions, tested, and my errors disappeared. I hope the PR https://github.com/PyCQA/astroid/pull/303 is acceptable.

@The-Compiler Thanks for very quick comment pointing exactly to where the issue came from.

**Comment by PCManticore** [comment]
Thanks @mbdevpl ! The PR looks fine, thank you for the first contribution. :D I'm going to merge it a bit later today.

**Comment by mbdevpl** [comment]
Actually, I made some silly mistakes in my first attempt... But now I re-pushed and it really is OK.

But still, the build is not passing, so I'm not sure if I destroyed the universe, or it was in ruin before I came here...

**Comment by PCManticore** [comment]
Ignore the test_builtin_fromlineno_missing failure, that's a long standing bug that no one got time to fix it and it's there to loom over us like Damocles's sword.
